assets/component-card.css

```diff
@@ -52,7 +52,6 @@
 .card--standard .card__inner:after {
   content: '';
   position: absolute;
-  z-index: -1;
```

Comment:
This has the biggest impact as it means, shadows aren't hidden in the back anymore but showing on top of the preceding element: 
<details> 
<summary>Screenshot</summary>

![](https://screenshot.click/17-57-kwbg2-7ef4f.png)
</details>

Reply:
This is outdated. Without animation the shadow isn't on top of the previous card anymore. It happens only when animation is on.